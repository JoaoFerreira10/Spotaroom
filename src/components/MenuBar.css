.navbar {
    background: #146A20 !important;
}

.navbar a:hover{
    color: #a2a2a2;
}