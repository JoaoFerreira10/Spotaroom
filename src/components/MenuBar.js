import React from 'react';
import './MenuBar.css';

class MenuBar extends React.Component{

    render () {
        return (<nav className="navbar fixed-top navbar-expand-lg navbar-dark">
                <a className="navbar-brand" href="#">spotaroom</a>
                <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
                  <span className="navbar-toggler-icon"></span>
                </button>
                <div className="collapse navbar-collapse" id="navbarNav">
                  <ul className="nav navbar-nav">
                    <li className="nav-item">
                      <a className="nav-link" href="#">The Company <span className="sr-only">(current)</span></a>
                    </li>
                    <li className="nav-item">
                      <a className="nav-link" href="#">How We Work</a>
                    </li>
                    <li className="nav-item">
                      <a className="nav-link" href="#">Contact Us</a>
                    </li>
                  </ul>
                </div>
              </nav>);
    }
}

export default MenuBar;