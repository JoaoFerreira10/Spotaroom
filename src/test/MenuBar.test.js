import React from 'react';
import { create } from 'react-test-renderer';
import MenuBar from '../components/MenuBar';

describe("MenuBar component test", () => {

  it("it matches with the snapshot", () => {
      
    const component = create(<MenuBar />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});