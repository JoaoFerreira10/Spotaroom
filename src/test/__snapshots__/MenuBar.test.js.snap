// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuBar component test it matches the snapshot 1`] = `
<nav
  className="navbar fixed-top navbar-expand-lg navbar-dark"
>
  <a
    className="navbar-brand"
    href="#"
  >
    spotaroom
  </a>
  <button
    aria-controls="navbarNav"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarNav"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="navbarNav"
  >
    <ul
      className="nav navbar-nav"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="#"
        >
          The Company 
          <span
            className="sr-only"
          >
            (current)
          </span>
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="#"
        >
          How We Work
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="#"
        >
          Contact Us
        </a>
      </li>
    </ul>
  </div>
</nav>
`;

exports[`MenuBar component test it matches with the snapshot 1`] = `
<nav
  className="navbar fixed-top navbar-expand-lg navbar-dark"
>
  <a
    className="navbar-brand"
    href="#"
  >
    spotaroom
  </a>
  <button
    aria-controls="navbarNav"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarNav"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="navbarNav"
  >
    <ul
      className="nav navbar-nav"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="#"
        >
          The Company 
          <span
            className="sr-only"
          >
            (current)
          </span>
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="#"
        >
          How We Work
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="#"
        >
          Contact Us
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
